pocs/camera/camera.py

```diff
@@ -56,13 +58,14 @@ class AbstractCamera(PanBase, metaclass=ABCMeta):
         firmware.  This can be done using ASICAP, or `pocs.camera.libasi.ASIDriver.set_ID()`.
     """
 
+    sub_classes = {'Focuser', 'FilterWheel'}
+    sub_names = {sub_class.casefold() for sub_class in sub_classes}
```

Comment:
Do these need to be public? Also, ask the risk of verbosity, I think a full `subcomonent_classes` and `subcomponent_names` might be more clear, especially if public.

Reply:
Also, why not just pass these in during initialization for those cameras that need them?

Reply:
It was a while ago, but I think my thinking here was that I wanted `Camera`s to always have all of these subcomponent attributes, but for them to be `None` if the corresponding hardware isn't present. I think this was motivated by a preference for `if self.<subcomponent>`: over
```
try:
    Do something with self.<subcomponent>
except AttributeError:
    pass
```
for conditional code that depends on the presence of subcomponents.

Reply:
Made private, and renamed.